doi-requests/build.gradle

```diff
@@ -37,5 +37,9 @@ dependencies {
 
 }
 
+test{
+    environment "AWS_REGION", "eu-west-1"
```

Comment:
Moving the default constructors of clients from the handlers, down to the service, requires more modules to be aware of this env variable.